.circleci/config.yml

```diff
@@ -133,12 +133,18 @@ references:
   upload_build_artifacts: &upload_build_artifacts
     run:
       name: Upload build artifacts to central repository (AWS)
-      command: aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+      command: |
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
```

Comment:
Let's have latest build as well, we'll probably use it on integration env setups